test/acceptance/features/bindAppToMultipleSbrs.feature

```diff
@@ -0,0 +1,54 @@
+@olm
+Feature: Bind a single application to multiple SBRs
+
+    As a user of Service Binding operator
+    I want to bind a single application to multiple SBRs
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+        * PostgreSQL DB operator is installed
+
+    Scenario: Bind a single db instance by creating 2 SBRs to a single application
```

Comment:
So, we speak about binding application to single service? Thus, this scenario should be moved to `bindAppToService.feature`

Also, let's describe better how this scenario is different from others we have already. From the applied steps, it looks like that we apply to the cluster two identical service bindings. Such action should be idempotent - the outcome should be equal to the scenario when just single service binding is applied.

Also, please consider replacing DB service with mocked one, and using generic test application, to make the test lighter. That would allow us to proper assert expectations, i.e. how application context looks like.